docs/content/user-manual/elasticjob-lite/configuration/built-in-strategy/error-handler.en.md

```diff
@@ -81,11 +81,11 @@ Maven POM:
 
 Configuration: 
 
-| Name                      | Description | Default Value     |
-| ------------------------- |:----------- |:----------------- |
-| webhook                   | TODO        | None              |
-| connectTimeoutMillisecond | TODO        | 3000 milliseconds |
-| readTimeoutMillisecond    | TODO        | 5000 milliseconds |
+| Name                             | Description                                                                | Default Value     |
+| -------------------------------- |:---------------------------------------------------------------------------|:----------------- |
+| wechat.webhook                   | The webhook address of the wechat robot                                    | None              |
+| wechat.connectTimeoutMillisecond | The timeout period for establishing a connection with the wechat server    | 3000 milliseconds |
+| wechat.readTimeoutMillisecond    | The timeout period for reading available resources from the wechat server  | 5000 milliseconds |
```

Comment:
How to understand "available resources"? Is that HTTP response?

Reply:
That is a HTTP response

Reply:
For wechat and dingTalk，we send the error message to a URL named webhook by HTTP POST method